src/main/java/com/thealgorithms/others/Fcfs.java

```diff
@@ -0,0 +1,93 @@
+package com.thealgorithms.others;
+
+import java.util.Scanner;
+
+public class Fcfs {
+    int[] completionTime;
+    int[] turnAroundTime;
+    int[] arrivalTime;
+    int[] burstTime;
+    int[] waitingTime;
+    private double avgWaitingTime;
+    private double avgTurnAroundTime;
+    int n;
+
+    public Fcfs(int size){
+        n = size;
+        completionTime = new int[n];
+        turnAroundTime = new int[n];
+        arrivalTime = new int[n];
+        burstTime = new int[n];
+        waitingTime = new int[n];
+        avgWaitingTime = 0;
+        avgTurnAroundTime = 0;
+        avgWaitingTime = 0;
+
+    }
+
+
+    public void printTable(){
+        System.out.printf("%s\t%s\t%s\t%s%n", "AT","BT","WT","TAT");
+        for(int i = 0; i < n; i++){
+            System.out.printf("%d\t%d\t%d\t%d%n", arrivalTime[i], burstTime[i], waitingTime[i], turnAroundTime[i]);
+        }
+    }
+    public double getAvgWaitingTime(){
+        double sum = 0;
+        double avg = 0;
+        for(int i = 0; i < n; i++){
+            sum += waitingTime[i];
+        }
+        avg = sum / n;
+        return avg;
+    }
+
+    public double getAvgTurnAroundTime(){
+        double sum = 0;
+        double avg = 0;
+        for(int i = 0; i <n; i++){
+            sum += turnAroundTime[i];
+        }
+        avg = sum / n;
+        return avg;
+    }
+
+    public static void main(String... args){
```

Comment:
Please put code in appropriately named methods, add JUnit tests and remove Hamming distance from this PR